ParkingDatabase/src/main/java/com/example/parkingdatabase/confiq/CarWebConfig.java

```diff
@@ -0,0 +1,28 @@
+package com.example.parkingdatabase.confiq;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.springframework.boot.web.server.WebServerFactoryCustomizer;
+import org.springframework.boot.web.servlet.server.ConfigurableServletWebServerFactory;
+import org.springframework.context.annotation.Bean;
+
+public class CarWebConfig{
+
+    ObjectMapper objectMapper = new ObjectMapper();
+    CarWebConfig carWebConfig = new CarWebConfig();
```

Comment:
Oh you are creating this class inside it. That's why you got problems. There is also no such method as setMessage()